web_client/views/imageViewerWidget/geojs.js

```diff
@@ -25,29 +25,32 @@ function guid() {
 
 var GeojsImageViewerWidget = ImageViewerWidget.extend({
     initialize: function (settings) {
-        ImageViewerWidget.prototype.initialize.call(this, settings);
-
         this._layers = {};
         this.listenTo(events, 's:widgetDrawRegion', this.drawRegion);
         this.listenTo(events, 'g:startDrawMode', this.startDrawMode);
         this._hoverEvents = settings.hoverEvents;
 
-        $.getScript(
-            staticRoot + '/built/plugins/large_image/extra/geojs.js',
-            () => {
-                this.trigger('g:beforeFirstRender', this);
-                this.render();
-            }
-        );
+        $.ajax({  // like $.getScript, but allow caching
+            url: staticRoot + '/built/plugins/large_image/extra/geojs.js',
+            dataType: 'script',
+            cache: true
+        }).done(() => {
+            this.trigger('g:beforeFirstRender', this);
+            ImageViewerWidget.prototype.initialize.call(this, settings);
```

Comment:
Making `initialize` call async breaks backbone conventions.  Is there a way to refactor this to keep the synchronous behavior?